### ISSUE TITLE
Competition: From the stone age to space

### ISSUE THREAD

**Issue by joe7575** [joe7575/techage_feature_requests#23]
Ich stelle mir einen Wettkampf vor, bei dem alle Teilnehmer gleichzeitig mit den gegebenen Startgegenständen (Erze, Werkzeug, Pflanzen,...) gegeneinander antreten. Wer als erster das Raumschiff erreicht, das weit oben am Himmel hängt, hat gewonnen und bekommt einen Preis. 
Jeder startet mit gleichen Voraussetzungen, Startpunkt für alle ist die Position auf der Erde unter dem Raumschiff.
Hunger ist deaktiviert, Kommandos wie /spawn auch. Das ganze findet weit draußen in einer "Wettkampf-Arena" statt, die rundherum protected ist und die danach auch für den nächsten Event wieder gelöscht wird. Je nach Startgegenständen kann die Zeit stark variiert werden.

Ziel ist es, mit Hilfe von TA1 - TA4 ein JetPack zu craften und damit zum Raumschiff zu fliegen.

Könnte ihr euch vorstellen, dass sowas funktioniert?

**Comment by PapaSmok** [comment]
Das einzige Problem, das ich sehe ist, dass Spieler, die grade viel Zeit haben, dabei deutlich im Vorteil sind.

**Comment by joe7575** [comment]
Ja, hier gäbe es 2 Möglichkeiten:
- alle gleichzeitig am WE (4 Stunden geplant, oder so)
- jeder wann er will (die Zeit wird gemessen und das Ergebnis auf einer Highscore Liste ausgegeben)

**Comment by acmgit** [comment]
Klingt nett, aber wäre das nicht mehr ein Feature für ein Game, als einen Mod?
Civilization lässt grüßen ;-).

Aber woran soll das gebaute Raumschiff gemessen werden?
5 Blöcke als Plattform oder ein komplexes Raumschiff mit Gärten, Kabinen, Navigation usw.?

**Comment by joe7575** [comment]
Ja, du hast Recht. Das ist kein Feature für techage, sondern eine Erweiterung für den Server. War von mir aber auch so gedacht.
Das Raumschiff wäre schon da, du musst es nur erreichen.

**Comment by acmgit** [comment]
Ok, dann kann ich es mir leichter vorstellen.
Eventuell wäre da [gravitisim](https://github.com/acmgit/gravitysim) hilfreich, daß ab gewisser Höhe die Gravitation ändert ;-).
Den Code habe ich von Gundul und in ein kleinen Mod gepackt

**Comment by maxhipp** [comment]
Find ich eine echt gute Idee, man könnte dafür zB eine Arena bauen, die sich nach dem Spiel idealerweise wieder in den Grundzustand zurückversetzt, in dieser Arena können alle herumlaufen und machen was sie wollen (natürlich möglichst am Gewinn arbeiten). Die Idee find ich aber echt cool!
